src/lib/Libwin/popen.c

```diff
@@ -218,7 +223,7 @@ win_popen(char	*cmd, const char *type, struct pio_handles *pio, struct proc_ctrl
 		return (1);
 	} else {
 		fprintf(stderr, "\npopen: CreateProcess() failed!\n");
-		errno = GetLastError();
+		log_err(-1, __func__, "failed in CreateProcess");
```

Comment:
the log_err(-1,) should be called immediately after failure. so please move this log to immediately after CreateProcess with if (!fsuccess) block